src/Medidata.ZipkinTracer.Core/Properties/AssemblyInfo.cs

```diff
@@ -37,7 +37,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.1.0")]
-[assembly: AssemblyFileVersion("1.1.0")]
-[assembly: AssemblyInformationalVersion("1.1.0")]
+[assembly: AssemblyVersion("1.1.1")]
+[assembly: AssemblyFileVersion("1.1.1")]
+[assembly: AssemblyInformationalVersion("1.1.1")]

```

Comment:
Can you bump the version number to 1.2.0? Although my previous PR (#74) already bumped it, I won't generate the package now therefore we can have this change as well in the 1.2.0 version.